pkg/handler/v2/metering/metering.go

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright 2021 The Kubermatic Kubernetes Platform contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metering
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/go-kit/kit/endpoint"
+
+	"k8c.io/kubermatic/v2/pkg/handler/middleware"
+	"k8c.io/kubermatic/v2/pkg/provider"
+	utilerrors "k8c.io/kubermatic/v2/pkg/util/errors"
+)
+
+func ListReportsEndpoint(userInfoGetter provider.UserInfoGetter) endpoint.Endpoint {
+	return func(ctx context.Context, request interface{}) (interface{}, error) {
+
+		// Todo(WeirdMachine): make this only available if metering is enabled
```

Comment:
Will this be:
- added before the release?
- available in the CE version?
- enabled by default for the EE version?
- possible to check if this feature is enabled/disabled so that we can also make it conditionally available in the UI?

Reply:
This should be included in the upcoming release and be available for EE. 
Enabling by default is not possible as the admin needs to provide some s3 configuration.
If this feature is enabled at all might be provided by #7448